src/server/raft_client.rs

```diff
@@ -174,7 +174,9 @@ impl Buffer for BatchMessageBuffer {
         let mut msg_size = msg.start_key.len()
             + msg.end_key.len()
             + msg.get_message().context.len()
-            + msg.extra_ctx.len();
+            + msg.extra_ctx.len()
+            // index: 3, term: 2, data tag and size: 3, entry tag and size: 3
+            + 11 * msg.get_message().get_entries().len();
```

Comment:
minor:  per entry length can be added in the loop below.
msg_size += entry.data.len() + 11; // index: 3, term: 2, data tag and size: 3, entry tag and size: 3

Reply:
Yes, but it's efficient to use multiply than loop.